hr_holidays_public/migrations/13.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Druidoo - Iván Todorovich <ivan.todorovich@druidoo.io>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        holidays_without_meeting = env["hr.holidays.public.line"].search(
+            [("meeting_id", "=", False)]
+        )
+        for holiday in holidays_without_meeting:
+            _logger.debug("Creating meeting for holiday: %s" % holiday.name)
+            holiday.meeting_id = env["calendar.event"].create(
+                holiday._prepare_holidays_meeting_values()
+            )
```

Comment:
a doubt (just for my learning) this is what causes that when installing or updating the module the first time the meetings for the existing public days are created?

Reply:
It's only when updating the module from a lower version, not on module install
https://odoo-development.readthedocs.io/en/latest/maintenance/data-migration.html